dev/com.ibm.ws.request.timing_fat/publish/files/server_timing_localOnly.xml

```diff
@@ -0,0 +1,37 @@
+<server description="new server">
+
+  <!-- Enable features -->
+    <featureManager>
+        <feature>jsp-2.2</feature>
+		<feature>jdbc-4.0</feature>
+		<feature>requestTiming-1.0</feature>
+    </featureManager>
+
+  
+	 <httpEndpoint id="defaultHttpEndpoint" host="*" />
+                   
+    <jdbcDriver id="DerbyEmbedded" libraryRef="DerbyLib"/>
+    <library filesetRef="DerbyFileset" id="DerbyLib"/>
+    <fileset dir="${shared.resource.dir}/derby" id="DerbyFileset" includes="derby.jar"/>
+	<javaPermission codebase="${shared.resource.dir}/derby/derby.jar" className="java.security.AllPermission"/>
+	<javaPermission codebase="derby.jar" className="java.security.AllPermission"/>
```

Comment:
This line is not needed, as the java permissions is already set in Line 16, for the derby.jar file